### PR TITLE
Expose stream info via MPRIS

### LINKED ISSUE #665: Feature request: Expose stream info via MPRIS
`cmus-remote -Q` shows stream metadata (see #23). It would be nice if this was exposed via MPRIS, too. If you consider this a reasonable feature, which naming convention would you suggest? Since there's no appropriate field in the [specs](https://www.freedesktop.org/wiki/Specifications/mpris-spec/metadata/), I would use something like "cmus:stream".

### PR DESCRIPTION
**Pull request by jolange**
... if a stream is currently playing

Resolves #665

### CODE REVIEW

**Comment by flyingmutant** [comment]
@mahkoh  thoughts on this one?

File: mpris.c [hunk 1]
```diff
@@ -385,6 +385,9 @@ static int mpris_metadata(sd_bus *_bus, const char *_path,
 		if (ti->discnumber != -1)
```
Comment (z3t0) [inline mpris.c]: Sorry if this is a silly question but what does stream here refer to ?

Comment (jolange) [inline mpris.c]: A mp3/ogg stream, e.g. web radio.

Comment (z3t0) [inline mpris.c]: ahh got it

**Comment by mahkoh** [comment]
Why is the property called `stream` when the function is called `stream_title`?

Such extensions also need to be documented.

**Comment by jolange** [comment]
I called it `stream` for consistency with the `cmus-remote -Q` output (see #23). But I personally don't mind a different naming like `stream_title`.

I'd be happy to document this, but I did not find any documentation of the MPRIS interface at all. Where would you like to see it?

**Comment by mahkoh** [comment]
stream_title sounds better. Documentation can go in cmus(1). You should probably create a new section that talks about mpris and then a subsection about extensions.

**Comment by jolange** [comment]
@mahkoh I changed the name of the field and added a man page section. Let me know if you think something essential is missing.

**Comment by mahkoh** [comment]
Merged

**Pull request closed without merging**